_episodes/01-intro-to-library-carpentry.md

```diff
@@ -0,0 +1,132 @@
+
+> ## Documenting your data
+> Whether you create a text README file, a data dictionary, a codebook, or a project notebook, it is important
+> to document your data. This includes things like:
+> - where your came from
+> - why you collected it
+> - the time period that the data refer to
+> - who collected and analyzed them
+> - the software you used
+> - information about what is in the columns and rows in your dataset
+> 
+> You can learn more about README files, data dictionaries, and codebooks, including examples, at the [University of Iowa Guide to Readme, Data Dictionaries, Codebooks](https://www.lib.uiowa.edu/data/manage/documenting/readme/).
+{: .callout}
+
+### Where?
+
+Where your data are stored is something that you might take for granted, but it is important to think through your storage options based on your sharing, backup, and security needs. While it can be convenient to use the default places your computer wants to save files, this can make it difficult to find them later and, if they are on your laptop or desktop hard drive, can cause hassles if your computer crashes and hasn't been backed up. It also makes it impossible for your colleagues to use the data unless you send it to them, which can mean that there are versions of your files out there that are not the most recent copy. Sharing files on a shared drive or cloud storage platform (like Box, DropBox or Google Drive) can help more people have access and provide a place to back up your files. 
+
+No matter where you are storing your data you should consider any security concerns, particularly when dealing with any data that might lead to the identification of your community members. If it is essential that you collect and store this information, check in with your local IT group to learn about which systems at your institution are safe for storing sensitive information.
+
+> ### Data about human subjects
+> 
+> Any time we collect and use data about other humans we have to think carefully about why, how, and where we are storing it.
+> Like so many systems that we use every day, our library and learning management systems are collecting increasing amounts of data about the people who use them, and we
+> need to respect their privacy. If you plan to use any data about your communities in presentations or publications it is a good idea to go through the process 
+> of filling out a request with your local Institutional Review Board (IRB) and going through the ethics training program. It is a great way to learn more about human
+> subjects research. 
```

Comment:
+ > subjects research. You should also check what legal requirements are relevant in your country (for example, GDPR in the European Union).